.gitlab/pipeline/build.yml

```diff
@@ -382,3 +382,21 @@ build-subkey-linux:
 #   after_script: [""]
 #   tags:
 #     - osx
+
+# bridges
+
+# we need some non-binary artifacts in our bridges+zombienet image
+prepare-bridges-zombienet-artifacts:
+  stage: build
+  extends:
+    - .docker-env
+    - .common-refs
+    - .run-immediately
+    - .collect-artifacts
+  before_script:
```

Comment:
Nit: Should we also create `/artifacts/bridges-polkadot-sdk/cumulus/scripts` here ? Not sure if needed. Just saw that it's missing.